src/main/antlr/SHRDataElementParser.g4

```diff
@@ -82,6 +83,7 @@ elementStringConstraint:    EQUAL STRING; // Can also be applied to fix a URI
 elementIntegerConstraint:   EQUAL MINUS? WHOLE_NUMBER; // Can also be applied to fix a decimal value
 elementDecimalConstraint:   EQUAL MINUS? WHOLE_NUMBER DOT WHOLE_NUMBER? EXP?;
 elementTypeConstraint:      (KW_SUBSTITUTE | KW_ONLY) (simpleOrFQName | primitive | tbd);
+elementSubsetConstraint:    KW_ONLY (subsetConstraint) (KW_OR (subsetConstraint))+;
```

Comment:
For future reference, the `(` `)` around `subsetConstraint` are unnecessary -- but no big deal.

Reply:
Thank you for letting me know.